### PR TITLE
feat(ai-proxy): support assistant api

### PR DESCRIPTION
**Pull request by sfwn**
#### What this PR does / why we need it:

support assistant api

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=573217&iterationID=12783&type=TASK)

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  ai-proxy support assistant api           |
| 🇨🇳 中文    |    ai-proxy 支持 assistant api          |

### CODE REVIEW

**Review by wangzhuzhen — COMMENTED** [review]
/lgtm

File: cmd/ai-proxy/conf/routes/assistant.yml [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+          directors:
+            - TransAuthorization
+            - RewriteScheme
+            - RewriteHost
+            - AddModelInRequestBody
+            - AddHeaders(OpenAI-Beta=assistants=v1)
+      - name: audit-after-llm-director
+      - name: finalize
+
+  - path: /v1/threads/{thread_id}/messages
```
[the remark this answers is not in the document]
Comment (sfwn) [reply cmd/ai-proxy/conf/routes/assistant.yml:146]: yes, so client should store it or use `/v1/threads/runs` API.

File: cmd/ai-proxy/conf/routes/routes.yml [hunk 2]
```diff
@@ -9,10 +9,10 @@ routes:
...
       - name: rate-limit
       - name: context
-      - name: body-size-limit
-        config:
-          maxSize: 102400
-          message: { "messages": [ { "role": "ai-proxy", "content": "问题超长啦, 请重置会话", "name": "ai-proxy" } ] }
+#      - name: body-size-limit
+#        config:
+#          maxSize: 102400
+#          message: { "messages": [ { "role": "ai-proxy", "content": "问题超长啦, 请重置会话", "name": "ai-proxy" } ] }
```
[the remark this answers is not in the document]
Comment (sfwn) [reply cmd/ai-proxy/conf/routes/routes.yml:15]: proxied server side (OpenAI, Azure) will limit it according to model type.

File: cmd/ai-proxy/conf/routes/assistant.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (wangzhuzhen) [reply cmd/ai-proxy/conf/routes/assistant.yml:146]: ok

File: cmd/ai-proxy/conf/routes/routes.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (wangzhuzhen) [reply cmd/ai-proxy/conf/routes/routes.yml:15]: ok

**Pull request merged**